backend/minors/ENE.yaml

```diff
@@ -0,0 +1,226 @@
+---
+type: Minor
+name: Sustainable Energy
+code: ENE
+degree:
+urls:
+- https://acee.princeton.edu/education/minor-in-sustainable-energy/?utm_source=education&utm_medium=moreinfo
+description: The future well-being of societies, the global economy, and the global environment will benefit greatly 
+from collaborative research into renewable energy, alternative fuels, advanced energy conversion and storage systems, 
+remediation of environmental degradation, technology transfer to developing countries, and prudent judgment on policies 
+to support sustainable energy technology. Innovations and inventions require multidisciplinary approaches and 
+entrepreneurship, as well as grounding in theory and practice, all topics that are not covered by a single department. 
+The Minor Program in Sustainable Energy (the Minor) offers an integrated set of core and elective courses, introducing 
+students to fundamental concepts in energy technologies and technological change, providing depth in specific fields of 
+interest from specific energy systems to policy approaches that further sustainable energy use, exposure to laboratory 
+practices and experiences, and setting the stage for further work in the field.  
+contacts:
+- type: Program Manager
+  name: Moira Selinka
+  email: mselinka@princeton.edu
+req_list:
+- name: Core Courses
+  min_needed: 1
+  max_counted: 1
+  completed_by_semester: 6
+  explanation: |-
+    One of the following core courses is required as a prerequisite: ECS 301 or EPS 302. Students who are 
+    unable to take either ECS 301 or EPS 302 may petition the directors for other courses that offer an 
+    introduction to the study of Europe from a similarly interdisciplinary point, such as HUM 216-217, 
+    HUM 218-219, or HIS 212/EPS 212.
+  course_list:
+  - MAE 228
+  - ENE 202
+- name: Electives
+  min_needed: 4
+  max_counted: 1
+  pdfs_allowed: 1
+  max_common_with_major: 2
+  explanation: |-
+    Students must take four additional courses related to energy systems, technology, and policy. Students 
+    may choose from a list of approved electives that is reviewed each year by the Minor director. The list 
+    is organized by the six research foci of the Andlinger Center so that students can gain depth of knowledge 
+    in a particular area of sustainable energy, but students are free to choose any four courses from this list 
+    following these guidelines. Upon enrollment in the Minor, students will meet with the director or program manager 
+    to plan a coherent set of electives that complements their major and interests. One study abroad course may be 
+    taken as an elective with prior Program approval. Students may petition the Program for a course to be approved 
+    as an elective prior to the start of the semester. Elective petitions will not be accepted after the start of the 
+    semester or after the course has been completed.
+  course_list:
```

Comment:
Be careful here since a req can only have either a `course_list` or a `req_list` with child reqs, but not both. Let's do "Electives" as the top req with two child reqs, one for 300-level or above and one for all electives